### ISSUE TITLE
Problems with showProjection.

### ISSUE THREAD

**Issue by MigueLugo** [prody/ProDy#1003]
Hi there. Working with the release v10.10, and using:
**showProjection**(mprot_ens,mprot_pca[:2],rmsd=True), I'm having two problems:

1. Two plots are plotted instead of one.

2. When using in the same ensamble, the following:
**calcProjection**(mprot_ens[0],mprot_pca[0,1],rmsd=True)

the reported coodinates do not correspond to the graph. 

Years ago it worked for me (I published with ProDy). So, was it changed?

Thanks

**Comment by jamesmkrieger** [comment]
Hi,

1. This problem may be due to the auto_show setting but I'm not sure. Are you getting an empty plot and then a second one with the projection in it? I get that when I have the following code with auto_show is set to True: 

```plt.figure()
showProjection(mprot_ens,mprot_pca[:2])```

If you run `confProDy(auto_show=False)` then that should fix it and not make a new figure in between. 

2. This one is due to a difference in the default values of the norm parameter (it's True in calcProjection but False in showProjection). If you set it then you should get the same values again. Probably you want norm=False seeing as you're using rmsd=True.

Let me know if that fixes it.

Thanks

**Comment by jamesmkrieger** [comment]
I just thought I'd share the 4 different behaviours in case that helps.

<img width="1104" alt="Screenshot 2020-04-17 at 13 24 46" src="https://user-images.githubusercontent.com/13259162/79569049-d4bc5a80-80ae-11ea-826a-d8997299b3c0.png">

**Comment by jamesmkrieger** [comment]
I think most users probably want rmsd=True, norm=False

**Comment by jamesmkrieger** [comment]
I proposed the above pull request to fix the second problem. 

As for the first problem, please confirm whether the auto_show setting fixes it.

**Comment by MigueLugo** [comment]
Hi man,

Thanks a lot for yor fast reply

And very helpful. Problem solved. Please, include about 'norm' in the
Reference Manual.

* Miguel Lugo*

**Comment by jamesmkrieger** [comment]
You're welcome. Will do.

**Comment by jamesmkrieger** [comment]
I think this issue is sorted too. It can be referred while closed for deciding on pull request #1004. We'll think about whether makes sense to change the defaults but the documentation fix will be done anyhow.
